values.yaml

```diff
@@ -2,6 +2,11 @@
 
 location: sandbox
 
+# This value is used as a knob to switch in between 2 repos(serverservice vs fleetdb) temporarily.
+# Will be removed once finishing migration.
+repo_knob:
```

Comment:
I like this idea. Putting our knobs to just enable or disable a service quickly all in the same place. I think john's work also has a "knob" to enable firmware-syncer, and my work with fleet-scheduler has the same. 

Maybe just call it "knobs" or something. Some commonly set variables could be here. But this could also cause confusion in the long run in finding where some things are set, so maybe it should just be with the other fleetdb stuff below? Id like more feedback on this.

Reply:
We can simplify by renaming it to "knobs" and keeping it in its current position at the beginning of the file. 
This is because it serves as a set of feature flags for the entire repository, and I will remove it once the migration is completed.